components/feature/prompts/src/main/java/mozilla/components/feature/prompts/PromptFeature.kt

```diff
@@ -155,6 +160,8 @@ class PromptFeature private constructor(
     private val creditCardPickerView: SelectablePromptView<CreditCardEntry>? = null,
     private val onManageCreditCards: () -> Unit = {},
     private val onSelectCreditCard: () -> Unit = {},
+    private val onManageAddresses: () -> Unit = {},
+    private val addressPickerView: SelectablePromptView<Address>? = null,
```

Comment:
It's looks like the parameters of `PromptFeature` are growing to accommodate new feature, I believe instead of introducing separate parameters to this already large list, we could follow the pattern of using a delegate, where we can have all the related address functionality and just having one extra parameter, similar to what we are doing for [ShareDelegate](https://github.com/mozilla-mobile/android-components/blob/main/components/feature/prompts/src/main/java/mozilla/components/feature/prompts/PromptFeature.kt#L146).